src/requirements.in

```diff
@@ -46,9 +48,11 @@ paramiko==2.7.2
 pysftp==0.2.9
 sshtunnel==0.1.4
 flask-openid==1.3.0
+pyodbc==4.0.32
 pytest==6.2.5
 pytest-helpers-namespace==2021.04.29
 pytest-env==0.6.2
 pytest-mock==3.6.1
-pytest-postgres==4.0.0
+pytest-postgresql==4.0.0
```

Comment:
Good catch! If forgot to change this when I switched the testing approach.

Reply:
Yes I was looking for pytest-postgres all over the Internet but could not find it. Eventually though well must be the one with version +'ql' because that was offered on pypi ;-)